packages/wonder-blocks-birthday-picker/src/components/birthday-picker.tsx

```diff
@@ -285,7 +306,7 @@ export default class BirthdayPicker extends React.Component<Props, State> {
                 placeholder={this.labels.month}
                 onChange={this.handleMonthChange}
                 selectedValue={month}
-                style={{minWidth, ...dropdownStyle}}
+                style={{minWidth, ...defaultStyles.input, ...dropdownStyle}}
```

Comment:
```suggestion
                style={[{minWidth}, defaultStyles.input, dropdownStyle]}
```

Reply:
Good call Juan, I updated the PR with those changes!